### PR TITLE
[ MOSIP-27859 ] Adding pkt_cr_dtimes and setting the packet date and time from metadata of the packet 

### PR DESCRIPTION
**Pull request by Khuddusshariff0022**
…fo of the packet-[MOSIP-27859]

[MOSIP-27859]: https://mosip.atlassian.net/browse/MOSIP-27859?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: registration-processor/pre-processor/registration-processor-packet-validator-stage/src/main/java/io/mosip/registration/processor/stages/packet/validator/PacketValidateProcessor.java [hunk 1]
```diff
@@ -160,6 +162,9 @@ public class PacketValidateProcessor {
 	@Value("${mosip.notificationtype}")
 	private String notificationTypes;
 
+	@Value("${mosip.registration.processor.datetime.pattern}")
+	private String dateformat="yyyy-MM-dd'T'HH:mm:ss.SSS'Z'";
+
```
Comment (sowmya695) [inline registration-processor/pre-processor/registration-processor-packet-validator-stage/src/main/java/io/mosip/registration/processor/stages/packet/validator/PacketValidateProcessor.java:167]: Remove the hardcoded default value

File: registration-processor/pre-processor/registration-processor-packet-validator-stage/src/test/java/io/mosip/registration/processor/stages/packet/validator/PacketValidatorStageTest.java [hunk 2]
```diff
@@ -34,7 +34,8 @@ public class PacketValidatorStageTest {
 	@Mock
 	MosipEventBus mosipEventBus;
 	@InjectMocks
-	private PacketValidatorStage packetValidatorStage = new PacketValidatorStage() {
+	private PacketValidatorStage packetValidatorStage = new PacketValidatorStage()
```
Comment (sowmya695) [inline registration-processor/pre-processor/registration-processor-packet-validator-stage/src/test/java/io/mosip/registration/processor/stages/packet/validator/PacketValidatorStageTest.java:37]: This class doesn't have changes only, please remove unnecessary spaces added

File: registration-processor/registration-processor-core/src/main/java/io/mosip/registration/processor/core/packet/dto/packetvalidator/PacketValidationDto.java [hunk 3]
```diff
@@ -13,3 +13,4 @@ public class PacketValidationDto {
 	private String packetValidatonStatusCode="";
 	private String packetValidaionFailureMessage = "";
 }
+
```
Comment (sowmya695) [inline registration-processor/registration-processor-core/src/main/java/io/mosip/registration/processor/core/packet/dto/packetvalidator/PacketValidationDto.java]: This class doesn't have changes only, please remove unnecessary spaces added

File: registration-processor/registration-processor-registration-status-service-impl/src/main/java/io/mosip/registration/processor/status/entity/RegistrationStatusEntity.java [hunk 4]
```diff
@@ -58,6 +69,8 @@ public class RegistrationStatusEntity extends BaseRegistrationEntity {
 	@Column(name = "upd_dtimes")
 	private LocalDateTime updateDateTime;
 
+
+
```
Comment (sowmya695) [inline registration-processor/registration-processor-registration-status-service-impl/src/main/java/io/mosip/registration/processor/status/entity/RegistrationStatusEntity.java:73]: This class doesn't have changes only, please remove unnecessary spaces added

File: registration-processor/pre-processor/registration-processor-packet-validator-stage/src/main/java/io/mosip/registration/processor/stages/packet/validator/PacketValidateProcessor.java (hunk 1 above)
Comment (Khuddusshariff0022) [reply registration-processor/pre-processor/registration-processor-packet-validator-stage/src/main/java/io/mosip/registration/processor/stages/packet/validator/PacketValidateProcessor.java:167]: Removed

File: registration-processor/pre-processor/registration-processor-packet-validator-stage/src/test/java/io/mosip/registration/processor/stages/packet/validator/PacketValidatorStageTest.java (hunk 2 above)
Comment (Khuddusshariff0022) [reply registration-processor/pre-processor/registration-processor-packet-validator-stage/src/test/java/io/mosip/registration/processor/stages/packet/validator/PacketValidatorStageTest.java:37]: removed the spaces.
.

**Pull request closed without merging**